File: 코틀린/Lv.0 옹알이(1).kt
//https://school.programmers.co.kr/learn/courses/30/lessons/120956?language=kotlin
class Solution {
    fun solution(babbling: Array<String>): Int {
        var answer: Int = 0
        var stack = ""
        val two = listOf("ye", "ma")
        val three = listOf("aya", "woo")
        for(word in babbling){
            stack = ""
            for(i in word){
                stack += i
                if((stack.length == 2 && two.contains(stack)) || (stack.length == 3 && three.contains(stack))){
                    stack = ""
                }
            }
            if(stack.isEmpty()){
                answer += 1
            }
        }
        
        return answer
    }
}

File: Lv.0 옹알이(1).py
#https://school.programmers.co.kr/learn/courses/30/lessons/120956?language=python3
def solution(babbling):
    answer = 0
    for word in babbling:
        stack = ""
        for i in word:
            stack += i
            if(len(stack) == 2 and stack in ("ye", "ma")) or (len(stack) == 3 and stack in ("aya", "woo")):
                stack = ""
        if not stack:
            answer += 1
    return answer
